### PR TITLE
DEVDOCS-3082 Update webhooks with new params from swagger

### PR DESCRIPTION
**Pull request by markcmurphy**
[DEVDOCS-3082](https://jira.bigcommerce.com/browse/DEVDOCS-3082)

Added new query params to GET /hooks and updated the meta properties that are returned.

### CODE REVIEW

File: reference/webhooks.v3.yml [hunk 1]
```diff
@@ -2725,15 +2806,33 @@ components:
...
-          description: Total number of items in the result set.
-        total_items:
+          description: Number of items on current page.
+        per_page:
           type: integer
-          description: Total number of items.
+          description: Maximum number of items per page.
+        current_page:
+          type: integer
+          description: Current page number of results.
```
Comment (ektravel) [inline reference/webhooks.v3.yml:2820]: This item description is a bit confusing. Does it pertain to the page number or the number of results per current page?

**Review by ektravel — COMMENTED** [review]
Added one suggestion.

File: reference/webhooks.v3.yml (hunk 1 above)
Comment (markcmurphy) [reply reference/webhooks.v3.yml]: Good call, that is confusing. I have just updated to "Current page number"

**Review by ektravel — APPROVED** [review]
Swagger Editor did not detect any errors.

**Pull request merged**